macrosynergy/download/transaction_costs.py

```diff
@@ -66,5 +78,16 @@ def download_transaction_costs(
     return dfd
 
 
+def get_fids(df: QuantamentalDataFrame) -> list:
+    def repl(x: str, yL: List[str]) -> str:
+        for y in yL:
+            x = x.replace(y, "")
+        return x
+
+    return list(
+        set(df["cid"] + "_" + df["xcat"].apply(lambda x: repl(x, AVAILABLE_XCATS)))
```

Comment:
Do you not need to keep the `ctype`? Also for the AVAILABLE XCATS it is `<ctype><cost>_<stat>`

Reply:
I think I have simply misnamed the variable;
It should be `AVAILABLE_CCATS` (contract categories) of the form:
`<ctype><cost>_<size>`
complementing `<cid>_<ctype><rtype>_<...`

Reply:
Example filter `"AUD_FXBIDOFFER_MEDIAN".replace("FXBIDOFFER_MEDIAN", "") => "AUD"`

Reply:
Correct output should be `AUD_FX`.